controllers/dbconfig.php

```diff
@@ -28,6 +29,20 @@ public function createView( $error, $dbSaid ) {
 
             require 'views/dbconfig/create.php';
         }
+        private function runMigrations(){
+            require_once 'models/migration.php';
+            $migrations = Migration::findAll();
+            $this->init();
+            foreach ( $migrations as $migration ){
+                try {
+                    require_once 'database/migration/' . $migration;
+                }
+                catch( MigrationException $ex ){
+                    // Maybe change this behaviour to revert changes till now or something...
+                    throw $ex;
+                }
+            }
+        }

```

Comment:
This function should not be part of the dbConfig controller, which is responsible for writing the database configuration file. Perhaps the migration controller is more appropriate?